.cirrus.default.yml

```diff
@@ -270,7 +270,7 @@ sonarqube_task:
     unzip jacoco.zip -d ${CIRRUS_WORKING_DIR}/org.sonarlint.eclipse.core.tests/target/
     source cirrus-env QA
     .cirrus/regular_mvn_build_deploy_analyze \
-      -P-deploy-sonarsource,-release,-sign -Dmaven.install.skip=true -Dmaven.deploy.skip=true \
+      -P-deploy-sonarsource,-release,-sign -Dmaven.install.skip=true -DskipTests -Dmaven.deploy.skip=true \
```

Comment:
For consistency, may I suggest using `maven.test.skip=true` 😇?

Reply:
We cannot use it, as it is for Tycho Surefire, not Maven :open_mouth: